### ISSUE TITLE
Categorization of 'human travel'

### ISSUE THREAD

**Issue by johnwjudkins** [ApolloDev/apollo-sv#199]
The class 'human travel' is a descendant of biological_process. However, the definition of biological_process contains the phrase "specifically pertinent to the functioning of integrated living units: cells, tissues, organs, and organisms". Not all human travel is necessary for the functioning of a living unit. Perhaps 'planned process' is a more suitable parent of 'human travel'.

**Comment by dillerm** [comment]
We classified 'human travel' this way because we strongly felt that it is a type of behavior and, as it turns out, 'behavior' is a subclass of 'biological_process' in GO. I think one can also argue that human travel does fulfill some sort of biological function, even if indirectly. For example, someone might go on vacation for social reasons or as a way of finding stress relief, both of which would involve indirectly realizing some biological function in that person.

**Comment by cstoeckert** [comment]
Based on the current definition of GO 'biological process', I disagree with 'human travel' which encompasses planned human travel is always biological 'behavior'. The current definition of biological process from GO is "A biological process represents a specific objective that the organism is genetically programmed to achieve. Biological processes are often described by their outcome or ending state, e.g., the biological process of cell division results in the creation of two daughter cells (a divided cell) from a single parent cell. A biological process is accomplished by a particular set of molecular functions carried out by specific gene products (or macromolecular complexes), often in a highly regulated manner and in a particular temporal sequence." Note that this is different from the definition used by Apollo-SV. I don't think it holds that 'human air travel' (http://purl.obolibrary.org/obo/APOLLO_SV_00000533) and other types of planned 'human travel' are the result of genetic programming. Furthermore, such human travel would be a planned process which is disjoint from biological process (at least as defined in OBI). I recommend that you consider clarifying the definition of human travel such that non-genetically programmed travel would be excluded.

**Comment by hoganwr** [comment]
How to differentiate human migration from migration of other species?  Is human migration really disjoint from animal migration?  All the way up to bfo:process?

**Comment by cstoeckert** [comment]
Perhaps a distinction can be made between innate behavior versus the realization of a (non-genetic) plan. The differentia can be that 'geographical migration of organism' does not include realization of a plan which is the concretization of a plan specification. Human travel in the normal sense is the concretization of a plan specification.

**Comment by cstoeckert** [comment]
@hoganwr I seem to recall that you agreed with the distinction between innate behavior versus the realization of a (non-genetic) plan (see comment above). Can human travel be moved from under biological process based on this distinction?

**Comment by dillerm** [comment]
I agree, but that's because I'm of the opinion at the moment that GO:behavior should not be a subclass of GO:biological_process, given the definition of the latter (to my knowledge, there is no such thing as behavior that is genetically pre-programmed; rather, what is genetically pre-programmed is the brain morphology that gives rise to behaviors). I think a better parent for 'human travel' would be something like 'human activity'.

**Comment by hoganwr** [comment]
We note that the Mental Functioning Ontology similarly extends GO's behavior class to include things that are adult human behaviors that involve planning and choice.  So we do not have a good idea of how to reparent human travel.  Please recommend an alternative behavior class to GO's behavior.

**Comment by hoganwr** [comment]
Also, we note the discussion thread here: https://github.com/obo-behavior/behavior-ontology/issues/101

Happy to own human behavior, not genetically programmed either in OMRSE or perhaps MFO would like it and then use it to reparent its behavior class hierarchy. If they do not wish to, then again, happy to do it in OMRSE.

Perhaps the label "emergent biological behavior" for the new class since patterns of human behavior are not pre-programmed (we all seem to want to avoid determinism to a significant degree).

here's a paper although I haven't investigated enough yet:

https://pubmed.ncbi.nlm.nih.gov/1403968/

The word 'biological' in the label differentiates it from other phenomena of "emergence" outside the biological realm, e.g., even in engineered artifactual systems.
